lib/impact/sound.js

```diff
@@ -287,7 +287,11 @@ ig.Music = ig.Class.extend({
 		this.currentTrack.currentTime = 0;
 	},
 	
-	
+	currentTime: function() {
+		if ( !this.currentTrack) { return; }
```

Comment:
currentTime: function() {
  return this.currentTrack && this.currentTrack.currentTime;
}